code/mobility-services/mobility-services.css

```diff
@@ -228,6 +237,53 @@ a.big-button-container {
 a.small-button {
   text-decoration: none;
 }
+/****************************************/
+/*** Dropdown Menu Buttons Navigation ***/
+/****************************************/
+
+/* hide custom menu if on mobile */
+@media (max-width: 799px) {
+  #tia-menu-list {
+    display: none;
+  }
+}
+
+/* hide knack menu buttons if on desktop */
+@media (min-width: 799px) {
+  #view_163 {
+    display: none;
+  }
+}
+
+/*list, list item, and button stylings*/
+#oa-menu-list {
+  border-bottom: 0 !important
+}
+
+#oa-menu-list li {
+  margin-right: .5em;
+  box-shadow: 0px 2px 4px 0px gray;
+}
+
+#oa-menu-list .kn-button a {
+  color: #163f6e;
+  font-size: 1.1em;
+  border-bottom: 0;
+  padding-top:3px;
+}
+
+.oa-dropdown-menu-list .kn-button {
+  border-radius: 0;
+}
+
+.oa-dropdown-menu-list li {
+  border: 0;
+  margin-right: 0 !important;
+} 
+
+.oa-dropdown-menu-list li:hover {
+  background-color: rgb(235,235,235);
```

Comment:
I use #ebebeb now to match the updated Knack styling of their menus instead of the rgb